forge_sync/assets.py

```diff
@@ -3,13 +3,11 @@
 import requests
 from datetime import datetime
 
-from dagster import asset, Output, AssetMaterialization, ResourceParam
-
-from .resources import MongoDBResource
+from dagster import asset
 
 
 @asset
-def get_cdi_data(context, db_conn: ResourceParam[MongoDBResource]):
+def selic(context):
```

Comment:
```suggestion
def selic(context) -> Dict[str, Any]:
```
Ele adiciona a typehint na UI pra ter contexto do tipo.